app/forms/alert_form.rb

```diff
@@ -0,0 +1,34 @@
+class AlertForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveModel::Serialization
+  include ActiveModel::Dirty
+
+  attribute :summary
+  attribute :description
+  attribute :user_count
+  attribute :investigation_url
+
+  validate :summary_validation
+  validate :description_validation
+
+  def default_summary
+    "Product safety alert: "
+  end
+
+  def default_description
+    "\r\n\r\n\r\nMore details can be found on the case page: #{investigation_url}"
+  end
+
+  def summary_validation
```

Comment:
```suggestion
private

  def summary_validation
```

I'd move the `default_` methods to private as well if you implement the getter override suggestion.